kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/testplugins/SaslPlainTermination.java

```diff
@@ -31,6 +32,10 @@
 public class SaslPlainTermination
         implements FilterFactory<SaslPlainTerminationConfig, SaslPlainTermination.PasswordVerifier> {
 
+    static {
+        PlainSaslServerProvider.initialize();
```

Comment:
the integration test Sasl termination didn't work at first. In the other tests I guess we have initialized this through some other path by starting a Kafka server or configuring a KafkaClient in a specific way. Without this there is no `Provider` installed for plain.